devops/jobs/CreateSandbox.groovy

```diff
@@ -225,6 +225,9 @@ class CreateSandbox {
                 booleanParam("payment",false,"Enable Payment MFE")
                 stringParam("payment_version","master","The repository version of the frontend-app-payment")
 
+                booleanParam("learning",false,"Enable Learning MFE")
```

Comment:
I think this can also accept a commitish, do you know if that's the case?  If not we should probably update the help text and name of all of these version parameters.

Reply:
Thanks, Adam!  I'm not sure- the instructions on the setup page (linked in description) say this change will give the option to toggle the learning MFE on/off in the CreateSandbox job configuration UI, and then I think it's the [changes in the Config repo](https://github.com/edx/configuration/pull/6388/files) that will allow a user to specify a branch version, but this is my first time through so I could be wrong.